examples/test_data_gen/construct_test_data_flow/utils.py

```diff
@@ -1,250 +1,74 @@
-from __future__ import annotations
-
-import typing as t
-import warnings
-from collections import defaultdict
-
-try:
-    from llama_index.schema import BaseNode
-except ImportError:
-    raise ImportError(
-        "llama_index must be installed to use this function. " "Please, install it with `pip install llama_index`."
-    )
+import json
+import re
 
 import numpy as np
 import numpy.testing as npt
-from langchain.embeddings import OpenAIEmbeddings
-from langchain.embeddings.base import Embeddings
-from langchain.prompts import ChatPromptTemplate
+import openai
```

Comment:
Please double check is there any packages used that is not installed by default. If there is any, please put them in requirements.txt